### ISSUE TITLE
Prefix and suffix for textfields alternatives should be better managed 

### ISSUE THREAD

**Issue by vvalentin-lucca** [LuccaSA/lucca-front#2859]
We need to replace the aria-label with an aria-hidden + u-mask I think.
